applications/openshift/registry/ocp_insecure_registries/rule.yml

```diff
@@ -0,0 +1,57 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: "Check if any insecure registry sources is configured"
+
+description: |-
+    The configuration <tt>registrySources.insecureRegistries</tt> determines the
+    insecure registries that the OpenShift container runtime can access for builds
+    and pods. This configuration setting is for accessing the configured registries
+    without TLS validation which could lead to security breaches and should be
+    avoided.
+
+rationale: |-
+    Insecure registries should not be configured, which would restrict the possibilities of
+    OpenShift container runtime accessing registries which cannot be validated.
+
+severity: medium
+
+identifiers:
+  cce@ocp4: CCE-86023-9
```

Comment:
Please remember to remove the identifier from the `shared/references/cce-redhat-avail.txt` file so it doesn't get reused by someone else (same for the other id)

Reply:
comment incorporated.

Reply:
I'm afraid not, only the CCE from the other rule was removed from `shared/references/cce-redhat-avail.txt` you also need to remove this rule.

Reply:
CCE ID was used in another commit which was merged to master, replaced with another ID and is removed from `cce-redhat-avail.txt`.